website/scaleway.erb

```diff
@@ -178,6 +178,9 @@
             <li>
               <a href="/docs/providers/scaleway/d/security_group.html">scaleway_security_group</a>
             </li>
+            <li>
+              <a href="/docs/providers/scaleway/d/image.html">scaleway_image</a>
```

Comment:
You need to add a deprecation message to [`scaleway_image`](https://github.com/terraform-providers/terraform-provider-scaleway/blob/master/scaleway/data_source_image.go) schema.